tests/test_config_section_global.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+
+"""
+Test validating of GlobalSection.
+"""
+
+from base import TestBase
+
+from virtwho.config import GlobalSection, str_to_bool, MinimumSendInterval, DefaultInterval
+from virtwho.log import DEFAULT_LOG_DIR
+
+# Values used for testing GlobalConfigSection
+GLOBAL_SECTION_VALUES = {
+    'debug': True,
+    'oneshot': 'true',
+    'print': 'false',
+    'wrong_bool_value': 'nein',
+    'configs': ['local_libvirt.conf'],
+    'log_file': 'my_custom.log',
+    'interval': '120'
+}
+
+
+class TestGlobalConfigSection(TestBase):
+    """
+    Test base for testing class GlobalSection
+    """
+
+    def init_global_config_section(self):
```

Comment:
This looks like it is run every time before.  Does VirtWho not use `unittest.Testcase`?  The `setUp` method in `TestCase` seems like a good place to put this functionality so you don't have to call it by hand each time.